src/plugins/intel_cpu/src/node.h

```diff
@@ -68,7 +68,7 @@ class PortConfigurator {
           constant(constant),
           inPlace(inPlace) {}
 
-    PortConfigurator(ov::intel_cpu::LayoutType blockedDescType,
+    PortConfigurator(ov::intel_cpu::LayoutType blockedDescType,  // NOLINT(google-explicit-constructor)
```

Comment:
We need to specify the reason for NOLINT

Reply:
Huge diff, increased code verbosity